apps/client/src/config/autofill.js

```diff
@@ -3,10 +3,10 @@ import { addQuotes, getRestrictionByTypeName } from "../utils";
 const getDataNeed = (checker) =>
   checker && checker.getAutofillDataNeeds(autofillMap)[0];
 
-export const getDataNeedPageOrNext = (checker, autofillRoutes, routes) => {
-  const dataNeed = getDataNeed(checker);
-  return dataNeed ? autofillRoutes[dataNeed][0] : routes.questions;
-};
+export const getDataNeedPageOrNext = (checker, autofillRoutes, routes) =>
+  getDataNeed(checker)
+    ? autofillRoutes[getDataNeed(checker)][0]
+    : routes.wrapper;
```

Comment:
This function was made to check which route to render before going into the checker and navigating back from questions to intro. In the new wrapper page this doesn't make any sense, since there are no additional pages. There are additional components like Location/Address stuff, but I'm not sure if you still need a similar function for that. It could be implemented in the stepper instead. Let's discuss.

Reply:
I think we should implement this in this on component level, but let's discuss